app/assets/stylesheets/_authors.scss

```diff
@@ -0,0 +1,9 @@
+.author__name{
+  font-size:1.5em;
+}
+
+.author__books{
```

Comment:
Selector `author__books` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space